pyflakes/checker.py

```diff
@@ -1557,6 +1598,15 @@ def CALL(self, node):
                 node.func.attr == 'format'
         ):
             self._handle_string_dot_format(node)
+
+        if (
+            _is_typing(node.func, 'cast', self.scopeStack) and
+            len(node.args) >= 1 and
+            isinstance(node.args[0], ast.Str)
+        ):
+            with enter_annotation(self):
+                self.handleNode(node.args[0], node)
```

Comment:
I've just spotted that we do in fact a way to get the parent of a node, do you think it would be neater to push this into the `STR` method and have it look at the parent, rather than doing this (which potentially processes the `STR` node twice)?

(I'm assuming that the parent of the `ast.Str` node would end up being the call func, but haven't tested that)

Reply:
I suspect that way would be more difficult to catch the `cast(str, 'not an annotation')` case -- this seems pretty striaghtforward as-is 🤷‍♂ -- up to you

Reply:
Ah, good point. Will leave as-is.